### DISCUSSION TITLE
Question about rumble.

### DISCUSSION THREAD

**Discussion by jsphonorio** [TeamOpenFIRE/OpenFIRE-Firmware#37] [Q&A] [answered]
I have been getting all my pieces together to build my first gun. I've been following a lot of the directions here. I followed the posted guide and video for the rumble.  But just realized that the rumble says any 5v motor but the Pico only has the 3.3 v. Did I miss a step or requirement for rumble?

**Comment by jsphonorio** [comment, marked as answer]
Looking at this 
"The 3V3(OUT) pin provides a 3.3V power output that can be useful to power external circuits. The VBUS (USB Power Input) is the micro-USB input voltage. So, if you're powering the Raspberry Pi Pico via the USB port using 5V, you'll get 5V on the VBUS pin."

I'm guessing the camera goes on the 3v3 out and I can attach the rumble to the v bus since I'll be powering it via USB?

**Reply by SeongGino** [reply]
Unless you're using a Wii camera which strictly requires 3.3V, a DF Robot camera can use either. But since you have 3.3V anyways, it doesn't matter but would be preferred for cleanliness.

Most rumble motors can run off 3.3V as well, either through `VSYS` or `3V3 OUT`, but if your motor *absolutely requires* 5V then you can use `VBUS` which pulls from the leftover voltage directly from the USB host's power.

**Comment by SeongGino** [comment]
Basically just answered yourself there, lol

**Reply by jsphonorio** [reply]
Thanks. Sorry about that. I was unsure and asked, and kept googling
